public/fonts/SplineSans_Complete/Fonts/WEB/README.md

```
@@ -0,0 +1,37 @@
+# Installing Webfonts
+Follow these simple Steps.
+
+## 1.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)